lib/cryptoexchange/exchanges/airswap/services/market.rb

```diff
@@ -0,0 +1,49 @@
+module Cryptoexchange::Exchanges
+  module Airswap
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super ticker_url
+          adapt_all output
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Airswap::Market::API_URL}/ticker"
+        end
+
+        def adapt_all(output)
+          output.map do |pair, output|
+            target, base = pair.split('_')
+            market_pair = Cryptoexchange::Models::MarketPair.new(
+                            base: base,
+                            target: target,
+                            market: Airswap::Market::NAME
+                          )
+          adapt(output, market_pair)
+          end
+        end
+
+        def adapt(output, market_pair)
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Airswap::Market::NAME
+
+          ticker.volume    = NumericHelper.to_d(output['baseVolume'])
+          ticker.last      = NumericHelper.to_d(output['last'])
+          ticker.high      = NumericHelper.to_d(output['high24hr'])
+          ticker.low       = NumericHelper.to_d(output['low24hr'])
+          ticker.timestamp = Time.now.to_i
```

Comment:
We no longer want to set Time.now.to_i for API that do not return time. Lets omit this